packages/web/components/ConnectedPage.tsx

```diff
@@ -39,6 +40,7 @@ export const ConnectedPage: React.FC<{
   }
 
   if (error) {
+    Honeybadger.notify(error);
```

Comment:
I think the right way to do this would be to create a generic errorHandler function in web/utils. Then import Honeybadger in that function (_in one place_). So if we want to switch to another error handling service we only have to change the code in one place.

Reply:
> I think the right way to do this would be to create a generic errorHandler function in web/utils. Then import Honeybadger in that function (_in one place_). So if we want to switch to another error handling service we only have to change the code in one place.

just checking - leaving the `HoneybadgerErrorBoundary` in `_app.tsx` is fine?

Reply:
> just checking - leaving the `HoneybadgerErrorBoundary` in `_app.tsx` is fine?

Yup!